vault/external_tests/token/token_test.go

```diff
@@ -28,16 +28,61 @@ func TestTokenStore_CreateOrphanResponse(t *testing.T) {
 
 	core := cluster.Cores[0].Core
 	vault.TestWaitActive(t, core)
-	client := cluster.Cores[0].Client
-
-	secret, err := client.Auth().Token().CreateOrphan(&api.TokenCreateRequest{
-		Policies: []string{"default"},
-	})
-	if err != nil {
-		t.Fatal(err)
+	priClient := cluster.Cores[0].Client
+	token := priClient.Token()
+
+	secClient := cluster.Cores[1].Client
```

Comment:
"sec" as a prefix in tests tends to refer to secondary clusters, not standbys.  Speaking of which: does this test really belong in oss?  The behaviour is going to be very different when it runs here vs ent, since oss standbys are going to be forwarding the http request body sight unseen, vs actually processing the request on ent with perf standbys.

Reply:
So, on the ENT side, I have added a test `TestPerfStandbyCreateToken` that practices creating service and batch tokens from a perfStandby. However, I felt the need for it here as well to make sure sending a request to a standby node would still work, and I have not broken any existing flows. What do you think?

Reply:
The thing is, standby nodes don't even look at the request.  They just bundle up the bytes coming in over the wire and send it to the active node to be parsed and processed.  Which makes it pretty much the same as the single-node use case.

Reply:
That is my understanding as well. It felt like it is worth making sure that is still happening. 
Do you think I should just revert the changes for this test, because I believe we already have a similar test on the ENT side?

Reply:
I'm uncomfortable having a test that behaves so differently between OSS and ENT (though I'm sure it's not the first such test.) If you want to keep it, maybe have it t.Skip if IsEnterprise, and add a comment explaining this?